### DISCUSSION TITLE
Simplepyble's connect() can also do pairing

### DISCUSSION THREAD

**Discussion by Marce68** [simpleble/simpleble#336] [Q&A] [answered]
In my application I need to connect and pair to a peripheral device, can I do this using a call like peripheral.connect()?

**Comment by eriklins** [comment, marked as answer]
Yes, there are function like connect(), disconnect(), etc. However, if pairing or bonding is wanted, then this would need to be done prior to connecting with SimpleBLE on the OS level (e.g., Windows Bluetooth & devices -> Devices, similar on Mac or Linux). It cannot be done with SimpleBLE alone. To my knowledge this is a restriction of the underlying OS's.
